go/runtime/host/sandbox/sandbox.go

```diff
@@ -369,18 +369,19 @@ func (r *sandboxedRuntime) startProcess() (err error) {
 	hi.LocalConfig = r.rtCfg.LocalConfig
 
 	// Perform common host initialization.
-	var rtVersion *version.Version
 	initCtx, cancelInit := context.WithTimeout(ctx, runtimeInitTimeout)
 	defer cancelInit()
-	if rtVersion, err = pc.InitHost(initCtx, conn, hi); err != nil {
+
+	rtVersion, err := pc.InitHost(initCtx, conn, hi)
+	if err != nil {
 		return fmt.Errorf("failed to initialize connection: %w", err)
 	}
 
-	if r.rtCfg.Components[0].IsRONL() {
+	if id := r.rtCfg.Components[0]; id.IsRONL() {
 		// Make sure the version matches what is configured in the bundle. This check is skipped for
 		// non-RONL components to support detached bundles.
```

Comment:
Since this version is now per-component, it would probably be fine to check for all components. But probably we need to leave it like this for backwards compatibility until all bundles are using per-component versions.

Reply:
Do ROFLs send you the right version in the `RuntimeInfoResponse`? Maybe we open a new issue for that.

Reply:
They do. Just that previously there was only one version in the manifest which corresponded to the RONL component version.